docs/pt/docs/advanced/path-operation-advanced-configuration.md

```diff
@@ -0,0 +1,224 @@
+# Configuração Avançada da Operação de Rota
+
+## operationId do OpenAPI
+
+/// warning | Aviso
+
+Se você não é um "especialista" no OpenAPI, você provavelmente não precisa disso.
+
+///
+
+Você pode definir o `operationId` do OpenAPI que será utilizado na sua *operação de rota* com o parâmetro `operation_id`.
+
+Você precisa ter certeza que ele é único para cada operação.
+
+```Python hl_lines="6"
+{!../../docs_src/path_operation_advanced_configuration/tutorial001.py!}
+```
+
+### Utilizando o nome da *função de operação de rota* como o operationId
+
+Se você quiser utilizar o nome das funções da sua API como `operationId`s, você pode iterar sobre todos esses nomes e sobrescrever o `operationId` em  cada *operação de rota* utilizando o `APIRoute.name` dela.
+
+Você deve fazer isso depois de adicionar todas as suas *operações de rota*.
+
+```Python hl_lines="2  12-21  24"
+{!../../docs_src/path_operation_advanced_configuration/tutorial002.py!}
+```
+
+/// tip | Dica
+
+Se você chamar `app.openapi()` manualmente, os `operationId`s devem ser atualizados antes dessa chamada.
+
+///
+
+/// warning | Aviso
+
+Se você fizer isso, você tem que ter certeza de que cada uma das suas *funções de operação de rota* tem um nome único.
+
+Mesmo que elas estejam em módulos (arquivos Python) diferentes.
+
+///
+
+## Excluir do OpenAPI
+
+Para excluir uma *operação de rota* do esquema OpenAPI gerado (e por consequência, dos sistemas de documentação automáticos), utilize o parâmetro `include_in_schema` e defina ele como `False`:
+
+```Python hl_lines="6"
+{!../../docs_src/path_operation_advanced_configuration/tutorial003.py!}
+```
+
+## Descrição avançada a partir de docstring
+
+Você pode limitar as linhas utilizadas a partir de uma docstring de uma *função de operação de rota* para o OpenAPI.
+
+Adicionar um `\f` (um caractere de escape "form feed") faz com que o **FastAPI** restrinja a saída utilizada pelo OpenAPI até esse ponto.
```

Comment:
Here it is a little strange not translating form feed. But I confess I can't find a good translation to it.

Let's see if anyone can help us, but for now, maybe you can leave the way it is 😄

Reply:
I kept "form feed", i saw the term "alimentação de formulário" being used in automated translations, but it leads to misguiding pages on web search, i didn't found any good translation for this specific term

Reply:
The purpose of translating is to provide the same content in another language, so it does not make sense to keep untranslated words alongside translated. It is confusing.

I recommend translating it because any science has its own words to describe things proper from that science.

Reply:
I found other sources on the web using the term "alimentação de formulário", so i will translate this phrase as `um caractere de escape para alimentação de formulário`